### PR TITLE
chore(components/progress): infinite progress refacto

### PR DESCRIPTION
**Pull request by ncomont**
**What is the problem this PR is trying to solve?**
Full CSS infinite progress animation is too complicated to debug and to maintain.

**What is the chosen solution to this problem?**
Infinite progress animation is now managed by an additional div. The width ratio of the white part can easily be modified.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Progress/Progress.component.js [hunk 1]
```diff
@@ -24,14 +24,15 @@ function Progress({ id, percent, tooltip, infinite, contained }) {
 		{
 			[theme.hidden]: normalizedPercent === 0,
 			[theme.fixed]: !contained,
+			[theme.infinite]: !!infinite,
```
Comment (jsomsanith) [inline packages/components/src/Progress/Progress.component.js]: You don't need `!!`

File: packages/components/src/Progress/Progress.component.js [hunk 2]
```diff
@@ -24,14 +24,15 @@ function Progress({ id, percent, tooltip, infinite, contained }) {
...
 		}
 	);
-	const progressionClassNames = classNames(
-		theme['progress-percent'],
-		{ [theme.infinite]: !!infinite }
-	);
 
-	let progress = <div style={style} className={progressionClassNames} />;
+	let progress = (
+		<div style={style} className={classNames(theme['progress-percent'])}>
```
Comment (jsomsanith) [inline packages/components/src/Progress/Progress.component.js]: You don't need classNames, you only compose 1 class
`className={theme['progress-percent']}`

File: packages/components/src/Progress/Progress.component.js [hunk 3]
```diff
@@ -24,14 +24,15 @@ function Progress({ id, percent, tooltip, infinite, contained }) {
...
+			{ infinite && <div className={classNames(theme['infinite-indicator'])} /> }
```
Comment (jsomsanith) [inline packages/components/src/Progress/Progress.component.js]: Same here

File: output/components.sasslint.txt [hunk 4]
```diff
@@ -5,10 +5,13 @@ Scoping to packages that match 'react-talend-components'
 > sass-lint -v -q
 
 
+src/Progress/Progress.scss
+  42:45  error  Include leading zeros on numbers  leading-zero
```
Comment (jsomsanith) [inline output/components.sasslint.txt]: Linter error : `.6s` --> `0.6s`

File: packages/components/src/Progress/Progress.scss [hunk 5]
```diff
@@ -29,37 +31,16 @@ $empty-gradient-part-width: 100% / 9;
...
-		}
-
-		&.infinite {
-			background: transparent;
-			width: 100%;
-			animation: infinite-progression-keyframes 1s infinite;
-			animation-timing-function: linear;
+		.infinite-indicator {
+			height: 100%;
+			background: white;
```
Comment (jmfrancois) [inline packages/components/src/Progress/Progress.scss]: please use $white

Comment (ncomont) [inline packages/components/src/Progress/Progress.scss]: Done, thanks

**Pull request merged**